odp/schema/tag/daterangeinc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$id": "https://odp.saeon.ac.za/schema/tag/daterangeinc",
+    "$schema": "https://jschon.dev/meta/translation/schema",
+    "title": "Date Range Increment",
+    "description": "Schema for a date range increment tag.",
+    "type": "object",
+    "properties": {
```

Comment:
I suggest reducing the schema down to what is minimally needed to support the use case. This means only an `end` property is needed and the only enum option is `current_date`. There should also be a `required` array adjacent to properties to make sure an empty dict cannot be supplied as tag instance data.

Thus, the packaging service can be sure that it caters to 100% of the ways that the tag instance can be created (for this tag there is exactly one way).